contracts/nft/src/events/mod.rs

```diff
@@ -0,0 +1 @@
+
```

Comment:
I'm not sure how this is planning to be used, but I wonder if it would be helpful to have some minimal set of events.

Reply:
Good call. finally got these cleaned up and pushed